library/std/src/sys/unix/time.rs

```diff
@@ -9,12 +9,24 @@ pub const UNIX_EPOCH: SystemTime = SystemTime { t: Timespec::zero() };
 pub const TIMESPEC_MAX: libc::timespec =
     libc::timespec { tv_sec: <libc::time_t>::MAX, tv_nsec: 1_000_000_000 - 1 };
 
-#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Copy, Clone, Eq, Ord, Hash)]
```

Comment:
This will lose `StructuralPartialEq` for `SystemTime`, right?

Reply:
oops yea, I'll fix that.